src/components/OurProject/components/LastProject/index.tsx

```diff
@@ -0,0 +1,18 @@
+import Slider from "react-slick";
+import { DataOurProject, settings } from "../../constants";
+import ItemLastProject from "./ItemLastProject";
+import "../../styles.css";
```

Comment:
sort import

Reply:
[here ](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/order.md)is import order of eslint. I follow it. 
is it wrong?